boardswarm-cli/src/main.rs

```diff
@@ -389,61 +408,71 @@ struct ConfigureArg {
 
 #[derive(Debug, Subcommand)]
 enum Command {
-    /// Configure boardswarm instances
+    /// Configure boardswarm instances access
     Configure(ConfigureArg),
     /// Retrieve the login information from the remote boardswarm server
     LoginInfo,
     /// Actuator specific commands
     Actuator {
+        /// Which actuator to use
```

Comment:
The convention that i seems to have been using has been `The X to` e.g:
```
$ boardswarm-cli  device write --help
Write data to a device volume

Usage: boardswarm-cli device <DEVICE> write [OPTIONS] <VOLUME> <TARGET> <FILE>

Arguments:
  <VOLUME>  The volume to write to
  <TARGET>  The volume target to write to
  <FILE>    Path to the file to write
```
Would be good to follow a consistent convention in all help messages